store/useStore.js

```diff
@@ -0,0 +1,12 @@
+import create from "zustand";
+import { lostItems as initialLostItems } from "../components/Db";
```

Comment:
why are you giving the variable a new name?

Reply:
DB.js is not an component i guess 😉

Reply:
> why are you giving the variable a new name?

For the sake of clarity, I think I will have to check this again in the end, so that all variables are clearly separated from each other and there are no bugs because of similar names.